### ISSUE TITLE
Running lnp-forge build multiple times fails

### ISSUE THREAD

**Issue by conchyliculture** [McArcady/lnp-forge#78]
cmake complains that add_subdirectory(twbt) is present multipletimes in ${dfhack_plugins_dir}/CMakeLists.custom.txt

This is due to the line:

```
echo "add_subdirectory(twbt)" >> ${dfhack_plugins_dir}/CMakeLists.custom.txt
```
in https://github.com/McArcady/lnp-forge/blob/master/scripts/build/twbt.sh#L36

When run multiple times, the CMakeLists.custom.txt file gets multiple lines of "add_subdirectory(twbt)" which cmale complains about.

Maybe consider replacing the ">>" with a single ">" ?

**Comment by McArcady** [comment]
Good catch!
fixed in 669da11

**Comment by McArcady** [comment]
I cancelled the patch.
Just 'clean' and 'build' again if it happens
